lib/multiple_search_selection.dart

```diff
@@ -633,7 +631,6 @@ class _MultipleSearchSelectionState<T>
     widget.onPickedChange?.call(pickedItems);
     widget.onItemRemoved?.call(item);
     setState(() {});
-    widget.onItemRemoved?.call(item);
```

Comment:
removing this line is the main thing being addressed, as it is called on 634 already and I don't think it's supposed to be called twice?